tests/performance/scale/tests/setup-env-var.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+set -eoux pipefail
+
+ELASTIC_USERNAME=$1
+ELASTIC_PASSWORD=$2
+
+export ELASTIC_USERNAME=$ELASTIC_USERNAME
```

Comment:
I'm not super familiar with the `source` command, but can we do: `export ELASTIC_USERNAME=$1`?